lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts

```diff
@@ -89,18 +89,19 @@ export class FormCloudService extends BaseCloudService {
         );
     }
 
-    createTemporaryRawRelatedContent(file, nodeId): Observable<any> {
+    createTemporaryRawRelatedContent(file, nodeId, contentHost): Observable<any> {
```

Comment:
can we add the types?

Reply:
for now, the type is not used and is always alfresco

Reply:
@eromano let me know if sounds good